FF151 Relnote/Doc: Sanitizer.replaceElementwithChildren() html/svg false (#43920)
mdn/content: files/en-us/web/api/sanitizer/replaceelementwithchildren/index.md

A boolean value: `true` if the operation updated the `Sanitizer` configuration to replace the element with its children, and `false` otherwise.

The method returns `false` if the sanitizer is already configured to replace the given element, or if the resolved element is [not allowed](#disallowed_replacement_elements).

> The method will return also return `false` if the following objects are passed:

Reviewer: this is probably a leftover?

Author: Doh, thanks very much.

> ## Description

Author: FYI decided to add as a description with the disallowed elements listed, because I'm not certain the list won't grow.

> The method will return also return `false` if the following objects are passed:
> 
> ## Description

Author: ```suggestion
The method returns `false` if the sanitizer is already configured to replace the given element, or if the resolved element is [not allowed](#disallowed_replacement_elements).

## Description
```

> The **`replaceElementWithChildren()`** method specifies an element that will be replaced by its child HTML elements when the sanitizer is used.

Reviewer: should "child HTML elements" be "child content" to cover both child elements and text content?

Author: Good point. What I have written might be OK, but I think it is worth being more clear. I've specified elements and nodes still..

```suggestion
The **`replaceElementWithChildren()`** method specifies an element that will be replaced by its child elements and/or text nodes when the sanitizer is used.
```

This is primarily used for stripping styles from text.

### Disallowed replacement elements

The following elements are not allowed as replacement elements:

- {{htmlelement("html")}} in the HTML namespace (`http://www.w3.org/1999/xhtml`).
- {{svgelement("svg")}} in the SVG namespace (`http://www.w3.org/2000/svg`).

> - {{mathmlelement("math")}} in the MATHML namespace (`http://www.w3.org/1998/Math/MathML`).

Reviewer: ```suggestion
- {{mathmlelement("math")}} in the MathML namespace (`http://www.w3.org/1998/Math/MathML`).
```

Author: Just FYI, I copied from the spec, but I prefer this too.

> The following method calls will all return `false` as the `Sanitizer` can't be updated.

Reviewer: ```suggestion
All the following method calls return `false` because the `Sanitizer` can't be updated for these elements:
```
